app/routes/events/utils.ts

```diff
@@ -62,9 +62,9 @@ export const EventTypeConfig: Record<EventType, ConfigProperties> = {
     textColor: '#FFF',
   },
   [EventType.KiD_EVENT]: {
```

Comment:
Please update all the constants and variables like these as well. No need to have "KiD" stuff in the codebase now that it's not a thing